token/core/zkatdlog/nogh/v1/driver/driver.go

```diff
@@ -125,7 +127,7 @@ func (d *Driver) NewTokenService(tmsID driver.TMSID, publicParams []byte) (drive
 	ip := ws.IdentityProvider
 
 	authorization := common.NewAuthorizationMultiplexer(
-		common.NewTMSAuthorization(logger, ppm.PublicParams(), ws),
+		common.NewTMSAuthorization(ctx, logger, ppm.PublicParams(), ws),
```

Comment:
We shouldnt have a context here. Conceptually the context is bound to a request. Here we are instantiating a service, which doesn't happen with every request.